OrbitCore/Capture.cpp

```diff
@@ -73,11 +73,12 @@ std::shared_ptr<Process> Capture::GTargetProcess = nullptr;
 std::shared_ptr<Session> Capture::GSessionPresets = nullptr;
 
 void (*Capture::GClearCaptureDataFunc)();
-void (*Capture::GSamplingDoneCallback)(
-    std::shared_ptr<SamplingProfiler>& a_SamplingProfiler);
 std::vector<std::shared_ptr<SamplingProfiler> > GOldSamplingProfilers;
 bool Capture::GUnrealSupported = false;
 
+Capture::SamplingDoneCallback Capture::sampling_done_callback_ = nullptr;
+void* Capture::sampling_done_callback_user_data_ = nullptr;
```

Comment:
What is the purpose of this user_data member?  A comment explaining the idea could be useful.

Reply:
done